js/functions.js

```diff
@@ -0,0 +1,68 @@
+/*
+Иван, добрый день! Подскажите, такой вариант первой функции будет верный? :
+const getLengthCheck = (string, maxLength) => { (string.length <= maxLength) ? true : false};
```

Comment:
Не совсем, ошибка в том, что сейчас функция, ничего не возвращает (будет `undefined`, как и указано ниже, т.е. функция просто выполняет сравнение и все, результат сравнения не используется, он уходит "вникуда", т.е. выделяется временно под него память и сразу же очистится при проходе сборщиком мусора), чтобы **стрелочная** функция возвращала значения и если она однострочная (как этот случай), то не нужно указывать скобочки тела этой функции, вот так:
```suggestion
const getLengthCheck = (string, maxLength) => (string.length <= maxLength) ? true : false;
// или явно возвращать результат выполнения сравнения
const getLengthCheck = (string, maxLength) => {
 return (string.length <= maxLength) ? true : false;
}
```

Более того, можно ещё упростить, результат выполнения сравнения `string.length <= maxLength` уже будет всегда булевым типом: true/false, т.е. условие if (в любом виде) тут избыточно, достаточно сделать так:
```suggestion
const getLengthCheck = (string, maxLength) => string.length <= maxLength;
```
Т.е. функция принимает 2 параметра, сравнивает длину первого аргумента со вторым агрументом и сразу же возвращает этот результат.